packages/teleterm/package.json

```
@@ -57,19 +57,22 @@
   },
   "productName": "Teleport Connect",
   "build": {
-    "appId": "gravitational.teleport.connect",
+    "appId": "com.gravitational.teleport.connect",
```

I suppose we should change this one too?

```suggestion
    "appId": "QH8AA5B8UP.com.gravitational.teleport.connect",
```

---

`build.mac.appId` will override `build.appId` when building for macOS. `bulid.appId` is going to be used on other platforms and I figured out we don't want to use Apple-specific IDs there.